website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -176,7 +176,7 @@ The "read" permission set is for users who need to view information about the st
 
 ## Organization Permissions
 
-Separate from project and workspace permissions, you can grant teams permissions to manage certain resources or settings across an organization. To set these permissions for a team, go to your organization's **Settings**. Then click **Teams**, and select the team name from the list.
+Separate from project and workspace permissions, you can grant teams permissions to manage or read certain resources or settings across an organization. To set these permissions for a team, go to your organization's **Settings**. Then click **Teams**, and select the team name from the list.
```

Comment:
_Quick, non-blocking nit-pick:_ I slightly prefer "view" over "read" here, but I don't feel strongly about this! 
```suggestion
Separate from project and workspace permissions, you can grant teams permissions to manage or view certain resources or settings across an organization. To set these permissions for a team, go to your organization's **Settings**. Then click **Teams**, and select the team name from the list.
```